website/docs/r/cognito_user_pool.markdown

```diff
@@ -92,6 +92,8 @@ The following arguments are supported:
 
 ~> **NOTE:** When defining an `attribute_data_type` of `String` or `Number`, the respective attribute constraints configuration block (e.g `string_attribute_constraints` or `number_attribute_contraints`) is required to prevent recreation of the Terraform resource. This requirement is true for both standard (e.g. name, email) and custom schema attributes.
 
+~> **NOTE:** Any changes to the schema attributes will re-create the User Pool.
```

Comment:
Nit: We typically omit notes like these since `terraform plan` will always have the most up-to-date difference handling and its easy to miss documentation updates like removing this should in-place update support be added in the future.